packages/exoflex/src/components/Switch.tsx

```diff
@@ -22,6 +22,29 @@ export type SwitchProps = AccessibilityProps & {
 
 const MARGIN = 2;
 
+export function getTrueWidth(
+  type: 'track' | 'thumb',
+  defaultWidth: number,
+  style?: StyleProp<ViewStyle>,
+): number {
+  let flattenedStyle = StyleSheet.flatten(style);
+  let width = flattenedStyle?.width;
+  let minWidth = flattenedStyle?.minWidth;
+  let maxWidth = flattenedStyle?.maxWidth;
+  let trueWidth = !!width
+    ? +width
+    : type === 'track'
+    ? defaultWidth
+    : defaultWidth / 2;
```

Comment:
This part is kinda weird, why do we divide the `defaultWidth` by 2 again?

I think we can simplify this part:
```ts
function getTrueWidth(type, defaultWidth, style) {
  if(type === 'track') {
    defaultWidth = defaultWidth / 2;
  }

  ...

  let trueWidth = isFinite(width) ? width : defaultWidth;
```

I'm still not sure about this though. Modifying the default width here feels wrong. Or maybe it's just the naming?